src/Compilers/CSharp/Test/Symbol/Symbols/StaticAbstractMembersInInterfacesTests.cs

```diff
@@ -23508,21 +23513,56 @@ void validate(ModuleSymbol module)
             }
         }
 
-        private static string ConversionOperatorName(string op) => op switch { "implicit" => WellKnownMemberNames.ImplicitConversionName, "explicit" => WellKnownMemberNames.ExplicitConversionName, _ => throw TestExceptionUtilities.UnexpectedValue(op) };
+        private static string ConversionOperatorName(string op, bool isChecked = false) =>
+            (op, isChecked) switch
+            {
+                ("implicit", _) => WellKnownMemberNames.ImplicitConversionName,
+                ("explicit", false) => WellKnownMemberNames.ExplicitConversionName,
+                ("explicit", true) => WellKnownMemberNames.CheckedExplicitConversionName,
+                _ => throw TestExceptionUtilities.UnexpectedValue(op)
+            };
+
+        private static string GetConversionOperatorName(string op, bool isChecked, out string checkedKeyword)
+        {
+            string opName = ConversionOperatorName(op, isChecked: false);
+            checkedKeyword = "";
+
+            if (isChecked)
+            {
+                string checkedName = ConversionOperatorName(op, isChecked: true);
+
+                if (checkedName == opName)
+                {
+                    return null; // There is no checked form
+                }
+                else
+                {
+                    opName = checkedName;
+                    checkedKeyword = "checked ";
+                }
+            }
+
+            return opName;
+        }
 
         [Theory]
         [CombinatorialData]
-        public void ImplementAbstractStaticConversionOperator_01([CombinatorialValues("implicit", "explicit")] string op, bool structure)
+        public void ImplementAbstractStaticConversionOperator_01([CombinatorialValues("implicit", "explicit")] string op, bool structure, bool isChecked)
         {
-            var typeKeyword = structure ? "struct" : "class";
+            string opName = GetConversionOperatorName(op, isChecked, out string checkedKeyword);
 
-            string opName = ConversionOperatorName(op);
+            if (opName is null)
+            {
+                return;
```

Comment:
Consider asserting which case this is.

Reply:
> Consider asserting which case this is.

Wouldn't this just duplicate the logic from GetConversionOperatorName?

Reply:
It would be helpful for me reading the test as a self-contained unit.